src/org/labkey/test/components/ui/files/FileUploadField.java

```diff
@@ -0,0 +1,187 @@
+package org.labkey.test.components.ui.files;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.react.MultiMenu;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+
+// AttachmentCard.tsx
+public class FileUploadField extends WebDriverComponent<FileUploadField.ElementCache>
```

Comment:
It looks like Trey is working on something similar:
https://github.com/LabKey/testAutomation/blob/00b6448b52fbfdeaa649befbbadde1f3d0f9002e/src/org/labkey/test/components/ui/files/AttachmentCard.java

I think it might make sense to include some of his public methods in this as well.

Because of the duplicate effort I'm going to hold off adding comments at this time until the three of us can resolve this.